spec/models/form_submission_spec.rb

```diff
@@ -11,4 +14,88 @@
   describe 'validations' do
     it { is_expected.to validate_presence_of(:form_type) }
   end
+
+  describe 'user form submission statuses' do
+    before do
+      @fsa = create(:form_submission, form_type: 'FORM-A', benefits_intake_uuid: SecureRandom.uuid, user_account:)
+      @fsb = create(:form_submission, form_type: 'FORM-B', benefits_intake_uuid: SecureRandom.uuid, user_account:)
+      @fsc = create(:form_submission, form_type: 'FORM-C', benefits_intake_uuid: SecureRandom.uuid, user_account:)
+
+      @fsa1 = create(
+        :form_submission_attempt,
+        benefits_intake_uuid: SecureRandom.uuid,
+        form_submission: @fsa,
+        created_at: 3.days.ago
+      )
+      @fsa2 = create(
+        :form_submission_attempt,
+        benefits_intake_uuid: SecureRandom.uuid,
+        form_submission: @fsa,
+        created_at: 2.days.ago
+      )
+      @fsa3 = create(
+        :form_submission_attempt,
+        benefits_intake_uuid: SecureRandom.uuid,
+        form_submission: @fsa,
+        created_at: 1.day.ago
+      )
+      @fsb1 = create(
+        :form_submission_attempt,
+        benefits_intake_uuid: SecureRandom.uuid,
+        form_submission: @fsb,
+        created_at: 1.day.ago
+      )
+    end
```

Comment:
This can be simplified by using `create_list`. I THINK you can use the `.zip` method (if I'm remembering my syntax correctly) to merge in the form_type's you want:

```suggestion
before do
  @fsa, @fsb, @fsc = create_list(:form_submission, 3, user_account: user_account).zip(['FORM-A', 'FORM-B', 'FORM-C']).map do |submission, form_type|
    submission.update(form_type: form_type, benefits_intake_uuid: SecureRandom.uuid)
    submission
  end

  create_list(:form_submission_attempt, 3, form_submission: @fsa) do |attempt, index|
    attempt.update(benefits_intake_uuid: SecureRandom.uuid, created_at: (3 - index).days.ago)
  end

  create(:form_submission_attempt, form_submission: @fsb, benefits_intake_uuid: SecureRandom.uuid, created_at: 1.day.ago)
end
```

Reply:
Oh yeah.  That's much better.  I'll change it.